src/poker/EquityEvaluator.java

```diff
@@ -8,16 +10,21 @@ public class EquityEvaluator {
     public final List<Card> hand2;
     Deck deck;
     private List<Card> board;
+    public  Player player1;
+    public Player player2;
 
     public HashMap<String, String> equityMap;
 
 
-    public EquityEvaluator(Deck deck, List<Card> hand1, List<Card> hand2, List<Card> board) {
+    public EquityEvaluator(Deck deck, List<Card> hand1, List<Card> hand2, List<Card> board, Player player1, Player player2) {
```

Comment:
tak jak patrze na ta klase to mozna by tutaj bylo troche uprzatnac patrz ponizej komenty

Reply:
Byniu ale jak wywaliles te zmienne globalne to konstruktor tez trzeba wyczyscic bo teraz trzeba podawac usles klasy przy tworzeniu new EquityEvaluator